Assets/Scripts/checkOneTimeTrigger.cs

```diff
@@ -18,10 +18,11 @@ public class checkOneTimeTrigger : MonoBehaviour {
 	}
 
 	public void UpdateTrigger () {
-		isTriggeredOnce = GameManager.getInstance ().objectTriggered[index];
+		hasBeenTriggeredOnce = GameManager.getInstance ().objectTriggered[index];
```

Comment:
So over here, your seeing if there is an object in the objecttriggered and once there is you set it true?